eng/testing/RunnerTemplate.cmd

```diff
@@ -88,7 +88,14 @@ echo ----- end %DATE% %TIME% ----- exit code %_exit_code% ----------------------
 if %_exit_code%==1 (
   if %HAS_TEST_RESULTS%==1 (
```

Comment:
I think we need a different case.  The dump scenario shouldn't set HAS_TEST_RESULTS.  I think you can test this locally by setting these environment variables.

Reply:
Looking at the code again, I should put my code inside the `if %_exit_code%==1` but after the check for `if %HAS_TEST_RESULTS%==1`. The dump file detection should happen only if the test exit code was not successful, but as you mentioned, should happen independently of HAS_TEST_RESULTS.

Reply:
I also found that the `_exit_code` is set to a very low negative value that the `IF %_exit_code% LSS 0` condition is unable to detect as true.